### PR TITLE
Pact Station

### PR DESCRIPTION
**Pull request by LukeZurg22**
# Pact-Station
Named after a pact struck between the self declared nations of Cargonia and Salvagistan
after revolting and fleeing from their old station home. Pact-Station was born from the
expansion of the two vessels that docked, with two major splits in design representing the priorities of either department.

Strangely, some designs of the station bled into the NanoTrasen style over time as the unique identity forged by Cargonia was lost. The original crew
that once maintained the station and called it home have long since dissapeared under mysterious circumstances.

## MAIN DIFFERENCES
- Heads of departments no longer are confined to their department, but share a communal quarters area in contrast to NanoTrasen's quarters arrangements.

- In security there is a Liberation Station (ammo-only as of current). This is mostly there to see how games tend to go with it there, and how Sec would handle having ammunition more readily-accessible. If it *proves* to be very damaging to the fun of the average round, then it'll be removed.

- Cargo sell palette is on-station in a personal Quartermaster-specific room below cargo.

- Perma isn't made to be miserable, its made to be escapable. People who grief might as well get smitten if they're going to make the game miserable for everyone but themselves.

- Hidden useful items about the station. Gives new players reason to explore, and returning players reason to step outside of their department for at least five seconds.

- Some syndicate-related stuff. Mostly harmless. Mostly...

- Secrets in many parts of the station + lore pieces. Some easier to get to than others.

- Research point generator but only one and with modest point generation because lets be honest, Science isn't fun when its grindy.

- Enemies in Salvage bay and Central Maintenance Area. Makes things slightly more challenging, and gives Salvage and Sec something to establish themselves upon like in the case of Engineering and AME + Solars.

- All of Maintenance not *readily* available with perfect accesses everywhere. Non-security will have to ""unlock"" by opening some blast doors to open up a pathway for easier travel.

- This station was brought to you by Union J, so you are legally allowed to dislike non-humans.

- The station possesses a few secrets. Find them.

## Areas
Atmospherics and Cryo
atmos_and_cryo

Bridge & Meeting Room
bridge

Cargo & Botnay
cargo_and_botnay

Central Area of Station
central_area

Engineering
engineering

EVAC & Docks
evac_and_docks

HoP and Eastern Half of Station
hop_and_eastern_half

Medical Bay & Gen. Department
medical_bay

Power & Gravity
power_grav

Science Department & Robotics
science_department

Security Sector
security

Perma
security_perma

West Maintenance Area
west_maintenance

### Station Map
The entirety of the station as per ingame map is depicted here. Note that some tiny areas have been deliberately not included in these screenshots. They contain secrets for the sake of the lore and fun of the map.
image

# Special Thanks To
Skippy : testing
heree : assistance & quality assurance

### CODE REVIEW

**Review by GroggleG — CHANGES_REQUESTED** [review]
You changed our sln for some reason, revert that and Ill run the checks.

**Comment by LukeZurg22** [comment]
Note: RTG's in bulk not included. They were there for testing purposes.
For the **.sln**, simply merge and reject the **.sln** changes.

**Comment by GroggleG** [comment]
> You changed our sln for some reason, revert that and Ill run the checks.

Your .sln is STILL different

**Comment by LukeZurg22** [comment]
> > You changed our sln for some reason, revert that and Ill run the checks.
> 
> Your .sln is STILL different

Third time: I STILL do not know how to remove the .sln difference from this commit! I would highly advise just discarding the changes to the .sln on merge!

File: Content.Benchmarks/MapLoadBenchmark.cs
Comment (GroggleG) [inline Content.Benchmarks/MapLoadBenchmark.cs:1]: Uoui was also added, the // is just so that if there are merge conflicts

**Comment by GroggleG** [comment]
I may or may not forcemap this in the future, letting the checks run.

File: SpaceStation14.sln
Comment (LukeZurg22) [inline SpaceStation14.sln:1]: Again, I have no idea why it pushed the **.sln**. I did a double take at the changes panel and didn't even see the **.sln** changes _there._ Discard the **.sln** modifications on merge if needed.

**Comment by LukeZurg22** [comment]
> I may or may not forcemap this in the future, letting the checks run.

I ran this locally on my machine with the Weasel fork. Map loaded perfectly without any problems or errors. I'd wager the checks are giving a false positive. Why -I cannot say.

**Comment by GroggleG** [comment]
yeah the build test is almost done, this sh-

**Comment by GroggleG** [comment]
what

**Comment by Froxxin** [comment]
Screenshot 2024-11-02 021843

**Comment by Froxxin** [comment]
Sorry didn't mean to close it lol

**Comment by LukeZurg22** [comment]
> Screenshot 2024-11-02 021843

Desperately trying to appease the envelop-hating deity in the machine. All mere mentions of the envelop are gone from the file, yet it craves its memory to be bleached from the threads of time.

**Pull request merged**